source/about/cloud-subscriptions.rst

```diff
@@ -12,14 +12,13 @@ Buy a Cloud subscription
 
 .. note::
 
-  Your credit card will be verified and you'll be charged immediately.
+  - Your credit card will be verified and you'll be charged immediately.
+  - A minimum of ten users is required.
```

Comment:
*seats

We're switching to refer to "seats" for licenses/subscriptions and "users" to refer to actual users. This is because customers can purchase more seats than there are users.